dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -147,4 +147,5 @@ http.proxyredirect.httpPort.desc=The (non-secure) port to redirect from. Incomin
 http.proxyredirect.httpsPort=HTTPS port
 http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTTP requests that use the HTTP port are redirected to this port.
 
-
+http.autoDecompression=Auto decompression
+http.autoDecompression.desc=Controls whether the HTTP Channel will automatically decompress incoming request body data.
```

Comment:
Suggestion:
Specifies whether the HTTP channel automatically decompresses incoming request body data.